packages/core/src/application.ts

```diff
@@ -50,6 +50,7 @@ export class Application extends Context {
       name,
       namespace: 'controllers',
       type: 'controller',
+      defaultScope: BindingScope.TRANSIENT,
```

Comment:
Do we have a test to verify this behavior?

Also since TRANSIENT is the default scope anyways, is it necessary to explicitly configure it here again? I am fine either way, just asking.

Reply:
I wanted to make it explicit.